addons/website/static/src/components/dialog/seo.xml

```diff
@@ -58,41 +40,109 @@
     </tr>
 </t>
 
-<t t-name="website.MetaKeywords">
-    <section>
-        <label for="website_meta_keywords">
-            Keywords
-        </label>
-        <div class="d-flex" role="form">
-            <div class="input-group w-auto">
-                <input t-model="state.keyword" type="text" class="form-control" t-att-placeholder="isFull ? 'Remove a keyword first' : 'Keyword'" t-att-readonly="isFull" maxlength="30" t-on-keyup="onKeyup"/>
-                <select title="The language of the keyword and related keywords."
-                        t-model="state.language" class="btn btn-outline-primary pe-5 form-select">
-                    <t t-foreach="languages" t-as="lang" t-key="lang[0]">
-                        <option t-att-value="lang[0]"><t t-esc="lang[2]"/></option>
-                    </t>
-                </select>
-                <button t-on-click="() => this.addKeyword(state.keyword)" t-att-disabled="isFull" class="btn btn-primary" type="button">Add</button>
+<t t-name="website.SeoChecks">
+    <section class="mt-3 oe_seo_checks">
+        <h4>
+            SEO Checks
+        </h4>
+        <hr/>
+        <t t-if="state.altAttributesMissing.length > 0">
+            <p><i class="fa fa-times text-danger me-3"></i><span t-out="state.altAttributesMissing.length"></span> alt attributes(s) missing on image(s)</p>
+            <t t-foreach="state.altAttributesMissing" t-as="img" t-key="img.src">
+                <div class="row align-items-center">
+                    <div class="col-2">
+                        <img class="i img-thumbnail" t-att-src="img.src"></img>
+                    </div>
+                    <div class="col-10" role="form">
+                        <div class="input-group w-auto">
+                            <input placeholder="Describe the image..." t-model="img.alt" type="text" t-att-class="'form-control ' + (img.alt.trim().length ? 'is-valid' : 'is-invalid')"/>
```

Comment:
Just a remark: I would use the `t-model.trim="..."` but I don't know why this makes the update on the value 'lazy' (similar to `t-model.lazy.trim`) :man_shrugging: